File: Selenium/Activities/Activity_1.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Chrome() as driver:
    driver.get("https://v1.training-support.net")
    driver.find_element(By.ID,"about-link").click()
    print(f"Print of title is {driver.title}")
    driver.quit()

File: Selenium/Activities/Activity_1.java
package Activities;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;

public class Activity_1 {
    public static void main(String[] args) {
        WebDriver driver = new ChromeDriver();
        driver.get("https://v1.training-support.net");
        driver.findElement(By.id("about-link")).click();
        System.out.println("The title of about us page is: " + driver.getTitle());
        driver.quit();

    }
}
